rdm/0011-creators-contributors-metadata.md

```diff
@@ -0,0 +1,292 @@
+- Start Date: 2019-10-22
+- RFC PR: [#11](https://github.com/inveniosoftware/rfcs/pull/11)
+- Authors: Tom Morrell, Guillaume Viger
+
+# Contributors / Creators Schema Definition
+
+## Summary
+
+This RFC defines the JSON schema field(s) holding the
+contributors, creators, authors... of a bibliographic record. Because defining
+the whole schema is a big effort, we have broken it off into constituent parts.
+
+
+## Motivation
+
+- As an InvenioRDM hoster, I want to benefit from a common and already defined
+  bibliographic schema, so that I don't have to make my own.
+- As an InvenioRDM hoster, I want to use a schema that is compatible with
+  DataCite schema (and others), so that I can inter-operate with other metadata
+  formats.
+- As an InvenioRDM developer, I want to code to a common metadata
+  schema, so that I don't repeat code and so that I benefit from and to others'
+  work.
+- As a researcher, I want to cite a record with its authors/creators,
+  so that my citation follows standards.
+- As a researcher, I want a DOI for my record which is only possible
+  if the equivalent `Creator` field is passed to the minting authority (DataCite).
+- As an InvenioRDM hoster, I want to provide a generic name field, so that
+  international authors and contributors can enter their full name in their own
+  way.
+- As an InvenioRDM hoster, I want to support person and organization identifiers,
+  so that I can disambiguate between entities.
+
+
+## Detailed design
+
+Creators and contributors can be people or organizations. Their schema structure
+will be reused. Their definition in the `"definitions"` section is:
+
+```json
+{
+  "definitions": {
+    "identifier": {
+      "type": "object",
+      "description": "A generic identifier.",
+      "additionalProperties": false,
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "description": "Kind of identifier.",
```

Comment:
For all new identifier types (ads, bioproject, biosample, ean8, ensembl, genome, gnd, hal, isni, pmcid, refseq, sra, uniprot), we should provide a description and example

e.g. 

DOI
Digital Object Identifier; a character string used to uniquely identify an object. A DOI name is divided into two parts, a prefix and a suffix, separated by a slash.
https://doi.org/10.1016/j.epsl.2011.11.037

Reply:
This can be provided out of band, right? (Not in the metadata record itself)

Reply:
Yeah, in documentation or a RFC.  DataCite has a good listing, and it would be nice to extend it.